ext/pcntl/tests/001.phpt

```diff
@@ -25,13 +24,11 @@ function test_exit_signal(){
 	print "\n\nTesting pcntl_wifsignaled....";
 
 	$pid=pcntl_fork();
-    
+
 	if ($pid==0) {
-		sleep(10);
-		exit;
+		posix_kill(posix_getpid(), SIGTERM);
 	} else {
 		$options=0;
-		posix_kill($pid, SIGTERM);
```

Comment:
this may be something else.
Wrong behavior in conjunction with 1 (or 10) seconds delay can't be explained by race condition.

This test failed on Travis, some time ago, because on some system raise() didn't work as expected. Now, this should be fixed in master.